src/util/object_factory_parameters.cpp

```diff
@@ -8,6 +8,7 @@ Author: Diffblue Ltd
 
 #include "object_factory_parameters.h"
 
+#include <solvers/strings/max_nondet_string_length_default.h>
```

Comment:
`solvers` is not the right module for this file (`util` shouldn't depend on `solvers`).
Consider moving the definition to `util/magic.h`

Reply:
Makes sense, moved